internal/models/models.go

```diff
@@ -218,6 +245,13 @@ type VerifyArtifactResponse struct {
 	Verified bool `json:"verified"`
 }
 
+// GetApiV1ArtifactsImageParams defines parameters for GetApiV1ArtifactsImage.
+type GetApiV1ArtifactsImageParams struct {
+	Image    string  `form:"image" json:"image"`
+	Username *string `form:"username,omitempty" json:"username,omitempty"`
+	Password *string `form:"password,omitempty" json:"password,omitempty"`
```

Comment:
passing passwords in URL query is generally considered insecure and is a bad practice, even with HTTPS